### PR TITLE
Update Documentation

### PR DESCRIPTION
**Pull request by dianzrong**
**What kind of change does this PR introduce?**

Bug fix

**Summary**

- made numbered list more consistent
- specified pip3 instead of pip as pip may not work right after downloading Python3.10
- remove typo from PR template

**Checklist**
* [x] My code follows the style guidelines of OpenAdapt
* [x] I have perfomed a self-review of my code
* [x] If applicable, I have added tests to prove my fix is functional/effective
* [x] I have linted my code locally prior to submission
* [x] I have commented my code, particularly in hard-to-understand areas
* [x] I have made corresponding changes to the documentation (e.g. README.md, requirements.txt)
* [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 **Checklist**
 * [ ] My code follows the style guidelines of OpenAdapt
-* [ ] I have perfomed a self-review of my code
+* [ ] I have performed a self-review of my code
```
Comment (KrishPatel13) [inline .github/pull_request_template.md:13]: 🚀

**Review by KrishPatel13 — APPROVED** [review]
@dianzrong Great job! I do not have to click `perfomed` to auto-fix :)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -4,10 +4,9 @@ We would love to implement your contributions to this project! We simply ask tha
 
 ## Code Style
 
-This project follows the [Google Python Style Guide](https://google.github.io/styleguide/pyguide.html).
+This project follows the [Google Python Style Guide](https://google.github.io/styleguide/pyguide.html), and for imports, we like to put put external imports in alphabetical order first, a new line, and then local imports.
```
Comment (abrichr) [inline CONTRIBUTING.md:7]: This line is not grammatically correct. Can you please change it to:

```
This project follows the [Google Python Style Guide](https://google.github.io/styleguide/pyguide.html), with the following modifications:
- imports are specified in alphabetical order
- built-in imports, external imports, and local imports are grouped together, in that order, separated by newlines
```

Comment (dianzrong) [reply CONTRIBUTING.md]: Addressed in [135de20](https://github.com/OpenAdaptAI/OpenAdapt/pull/305/commits/135de2083414fc3f1e23ed82aa49f662f0a84b65)

**Pull request merged**